extract_stats_from_video.py

```diff
@@ -214,31 +222,35 @@ def process_video(input_video_path, args):
 
   matcher = Matcher()
   tracker_by_id = {}
+ 
+  
   # loop over the frames.
   total_frames = -1
   while True:
     total_frames += 1
     _, frame = vs.read()
     if frame is None:
       break
-
     frame = imutils.resize(frame, width=args["frame_width"])
+    # frame = imutils.resize(frame, width=video_witdh)
     if image_rect is not None:
       frame = utils.crop_image(frame, image_rect)
-
+    
+    # add black rect
+    frame = cv2.rectangle(frame, (0,3),(188,200),(0,0,0),-1)
     if total_frames % args["frame_skip"] == 0:
       predictions_rect = extract_rectangles_from_predictions(
           frame, dataset_name, detector_model, args["confidence"])
       tracker_by_id = matcher.match(tracker_by_id, predictions_rect, frame)
     else:
       predictions_rect = []
       for tracker in tracker_by_id.values():
```

Comment:
Can you split to something like:
for tracker, label in tracker_by_id.values()

OK?

Reply:
of course, i'll change that